pkg/signal/trap_linux_test.go

```diff
@@ -0,0 +1,65 @@
+// +build linux
+
+package signal
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTrap(t *testing.T) {
+	sigmap := map[string]os.Signal{
+		"TERM": syscall.SIGTERM,
+		"QUIT": syscall.SIGQUIT,
+		"INT":  os.Interrupt,
+	}
+	wd, _ := os.Getwd()
+	testHelperCode := wd + "/testfiles/main.go"
```

Comment:
This variable contains a path to a file, not code, so I think `s/testHelperCode/testBinaryPath/` would be more appropriate.